grpc/server/test/action_service_impl_test.cpp

```diff
@@ -0,0 +1,274 @@
+#include <gmock/gmock.h>
+#include <string>
+
+// TODO remove this include
```

Comment:
I would be in favor of moving `ActionResult` in its own file, just like I did with `ConnectionResult`.

Reply:
I like it if it's namespace inside the module but up to you.

Reply:
I would put it in `dronecore::action::ActionResult`, but in another file. Is that what you mean with the namespace inside the module?